packages/zosfiles/__tests__/__system__/cli/delete/uss/uss.files.delete.directory.system.test.ts

```diff
@@ -0,0 +1,152 @@
+/*
```

Comment:
These two test files deviate slightly from the pattern we used for the other commands. Other test file names begin with `cli.` and the rest of the filename mimics the command structure e.g. `cli.files.delete.uss.directory.system.test.ts`. Something similar to that would be good. Good job on the tests though, they all passed for me and look good.

Reply:
Ah yes. That's makes sense.